apps/src/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditor.jsx

```diff
@@ -0,0 +1,196 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {
+  programmingExpressionShape,
+  programmingEnvironmentShape
+} from '@cdo/apps/lib/levelbuilder/shapes';
+import color from '@cdo/apps/util/color';
+import Dialog from '@cdo/apps/templates/Dialog';
+import {connect} from 'react-redux';
+import {removeProgrammingExpression} from '@cdo/apps/lib/levelbuilder/lesson-editor/programmingExpressionsEditorRedux';
+import * as Table from 'reactabular-table';
+import {lessonEditorTableStyles} from './TableConstants';
+
+const styles = {
+  search: {
+    paddingBottom: 10
+  },
+  actionsColumn: {
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    backgroundColor: 'white'
+  },
+  remove: {
+    fontSize: 14,
+    color: 'white',
+    background: color.dark_red,
+    cursor: 'pointer',
+    textAlign: 'center',
+    width: '100%',
+    lineHeight: '30px'
+  },
+  dropdown: {
+    width: '100%'
+  }
+};
+
+class ProgrammingExpressionsEditor extends Component {
+  static propTypes = {
+    programmingEnvironments: PropTypes.arrayOf(programmingEnvironmentShape)
+      .isRequired,
+
+    // Provided by redux
+    programmingExpressions: PropTypes.arrayOf(programmingExpressionShape)
+      .isRequired,
+    removeProgrammingExpression: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      confirmRemovalDialogOpen: false,
+      programmingExpressionForRemoval: null,
+      programmingEnvironmentId: 0
```

Comment:
I don't see where this is used. Am I missing it?

Reply:
Probably a miss from pulling apart the other PR. I can remove